appinventor/lib/blockly/src/core/blockly.js

```diff
@@ -453,8 +446,8 @@ Blockly.onKeyDown_ = function(e) {
     }
     if (e.keyCode == 86) {
       // 'v' for paste.
-      if (Blockly.clipboard_) {
-        Blockly.mainWorkspace.paste(Blockly.clipboard_);
+      if (window.parent.Blocklies.clipboard_) {
```

Comment:
@evanrthomas Why are you using window.parent.Blocklies instead of Blockly for the clipboard_?

I don't believe this is the correct way to do this. We happen to put blockly in a frame, but you are introducing that directly into blockly core, which doesn't make that assumption.